### PR TITLE
Unhide 1.4 features

### LINKED ISSUE #1616: Unhide planned Spin 1.4 features
* CLI: Unhide `spin up --sqlite` flag
* Rust SDK: Move `sqlite` out of `experimental` feature
* CLI: Unhide `spin doctor` command

### PR DESCRIPTION
**Pull request by itowlson**
Fixes #1616.

### CODE REVIEW

**Review by fibonacci1729 — APPROVED** [review]

File: sdk/rust/src/lib.rs [hunk 1]
```diff
@@ -8,8 +8,7 @@ pub mod outbound_http;
 /// Key/Value storage.
 pub mod key_value;
 
-/// Sqlite
-#[cfg(feature = "experimental")]
```
Comment (rylev) [inline sdk/rust/src/lib.rs:12]: The feature is still being turned on in some places. While this won't break anything, we might want to clean up those uses in the rest of the codebase.

**Review by rylev — CHANGES_REQUESTED** [review]

File: sdk/rust/src/lib.rs (hunk 1 above)
Comment (itowlson) [reply sdk/rust/src/lib.rs:12]: Thanks for the catch - fixed

File: crates/trigger/src/cli.rs [hunk 2]
```diff
@@ -107,8 +107,9 @@ where
     #[clap(long = "key-value", parse(try_from_str = parse_kv))]
     key_values: Vec<(String, String)>,
 
-    /// Run a sqlite migration against the default database
-    #[clap(long = "sqlite", hide = true)]
+    /// Run a SQLite statement such as a migration against the default database.
+    /// To run from a file, prefix the filename with @ e.g. spin up --sqlite @migration1.sql
```
[the remark this answers is not in the document]
Comment (itowlson) [reply crates/trigger/src/cli.rs]: I was kind of thinking that a migration system might create separate files for each migration but yeah I've changed it.

**Comment by itowlson** [comment]
@rylev This is blocked on your request for changes - what is still needed to get it over the line please?

**Comment by rylev** [comment]
Looks good!

**Review by rylev — APPROVED** [review]

**Pull request merged**